### PR TITLE
Bathroom privacy and more

### PR DESCRIPTION
**Pull request by Sindorman**
It is weird to have cameras in each single bathroom on the station, especially since it lets people peak into shower if door is open. This can also add edge to the antags.

- Removes cameras from each bathroom on the station, except Brig prison bathroom.

Fixes the following issues:

- Unconnected Scrubbers pipe manifold located at 128,87,4 (the Kitchen)
- Unconnected Scrubbers pipe manifold located at 131,155,4 (the Security - Security Office)
- Unconnected Scrubbers pipe manifold located at 131,153,4 (the Security - Brig)
- Unconnected pipe located at 81,167,3 (the Engineering - Atmospherics)
- Unconnected pipe located at 82,167,3 (the Engineering - Atmospherics)
- Unconnected Air supply pipe located at 106,68,4 (the Cargo Lobby)
- Unconnected Air supply pipe located at 106,66,4 (the Cargo Mail Room)
- Unconnected Scrubbers pipe located at 130,87,4 (the Kitchen)
- Unconnected Air supply pipe located at 139,126,4 (the Medical - Exam Room 1)
- Unconnected Scrubbers pipe located at 139,126,4 (the Medical - Exam Room 1)
- Unconnected Air supply pipe located at 139,124,4 (the Medbay Hallway - Port)
- Unconnected Scrubbers pipe located at 139,124,4 (the Medbay Hallway - Port)
- Airlock Mining Processing Hatch with bad turf at (108,51,4) in Mining Refinery.
- Airlock Pre-Op Maintenance Access with bad turf at (129,110,4) in the Medbay Hallway - Port.
- Airlock Engineering External Access with bad turf at (149,92,4) in Medbay Maintenance.

Mapmerge deleted 343 unused keys

### CODE REVIEW

**Comment by Sindorman** [comment]
!review

File: html/changelogs/Sindorman-cameras.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "As per privacy update all bathrooms on the station no longer have cameras installed in them, with the exception of brig(prison) bathroom."
```
Comment (HeyBanditoz) [inline html/changelogs/Sindorman-cameras.yml]: ...of the brig's bathroom. I'm pretty sure most know the prison is actually the brig, and aren't _exactly_  the same.

Comment (Sindorman) [reply html/changelogs/Sindorman-cameras.yml]: There are two bathrooms in brig - one for security personally and one in the perma.

**Comment by Sindorman** [comment]
Accidentally closed it

**Pull request merged**